### ISSUE TITLE
Asche issue

### ISSUE THREAD

**Issue by CalmKhaos** [AlderArts/foe#279]
Talking with Asche about Fortune telling seems to lock in those options. I've tried clicking on 'Never mind' to go back to the previous set and seem to be unable to do so.

**Edit**

Oh, if you don't think it's too much trouble, I have found some italic run-ons in her dialogues. I'll paste them here if that's ok. I'll wait on your word first.

**Comment by aldergames** [comment]
Throw the italics in a separate issue. I'll look into the locking.

**Comment by CalmKhaos** [comment]
Understood.
